packages/polymarket-notifier/src/polymarketNotifier.js

```diff
@@ -76,7 +76,13 @@ class PolymarketNotifier {
       .map((contract) => {
         // excluding proposals without a proposed price and the requester is not Polymarket
         // the threshold for accepting a proposal is valid is currently set to 0.95 but can be adjusted
-        if (!contract.timestamp || !contract.proposedPrice || contract.requester != polymarketContract) {
+        if (
+          !contract.timestamp ||
+          !contract.proposedPrice ||
+          !contract.outcome1Price ||
+          !contract.outcome2Price ||
```

Comment:
You might want to be careful with using `!` on a value that could be a number or undefined. If that number were ever _exactly_ 0, this would evaluate to true, despite it being defined.

For detecting that a Number is missing, prefer `contract.outcome1Price === undefined` to ensure that you _only_ capture undefined.

timestamp and proposedPrice are safer because the former should never be 0 and proposedPrice is a string that should never be an empty string (which would also evaluate to true here).

Example:
```
> !0
true
> !""
true
```

Reply:
Really good point. I updated to only capture undefined values. Thanks for the feedback.